server/routes/avatar/room.js

```diff
@@ -41,7 +42,7 @@ export const roomAvatar = Meteor.bindEnvironment(function(req, res/* , next*/) {
 		return;
 	}
 
-	const svg = renderSVGLetters(roomName, req.query.size && parseInt(req.query.size));
+	const svg = renderSVGLetters(roomName, req.query.size && parseInt(req.query.size), avatarColor);
```

Comment:
I would pass this as an extra object parameter.
```suggestion
	const svg = renderSVGLetters(roomName, req.query.size && parseInt(req.query.size), { avatarColor });
```